plugins/services/src/js/reducers/serviceForm/JSONReducers/__tests__/UnknownVolumes-test.js

```diff
@@ -0,0 +1,137 @@
+const { ADD_ITEM } = require("#SRC/js/constants/TransactionTypes");
+const UnknownVolumes = require("../UnknownVolumes");
+
+describe("UnknownVolumes", function() {
+  describe("#JSONParser", function() {
```

Comment:
Following http://www.betterspecs.org/#contexts extracting a common context which is `with empty input` should be done via contexts or nested describe blocks in our case. While it seems not making any sense in this particular example it draws the line between `with empty input` and `with any other inputs` that can have multiple expectations as well as multiple contexts.